docs/devel/releasing.md

```diff
@@ -257,8 +257,8 @@ been automated that need to happen after the branch has been cut:
 *Please note that this information may be out of date.  The scripts are the
 authoritative source on how version injection works.*
 
-Kubernetes may be built from either a git tree (using `hack/build-go.sh`) or
-from a tarball (using either `hack/build-go.sh` or `go install`) or directly by
+Kubernetes may be built from either a git tree (using `make`) or
+from a tarball (using either `make` or `go install`) or directly by
 the Go native build system (using `go get`).
```

Comment:
I doubt the last clause is true anymore. certainly a version built without autogenerated files will not be performant.